src/GeositeFramework/plugins/launchpad/main.js

```diff
@@ -86,6 +86,33 @@ define([
                 }));
 
                 $(this.container).empty().append($el);
+
+                // add truncate listeners to each toggle botton
+                $(".truncate-container").each(function () {
+                    if (parseFloat($('p', this).css("height")) > 50) {
+                        $('p', this).each(function() {
+                            this.classList.toggle('truncate-box');
+                        });
+
+                        this.addEventListener('click', function(ev) {
+                            if (ev.target.tagName === 'BUTTON') {
+                                $('p', this).each(function() {
+                                    this.classList.toggle('truncate-box');
+                                });
+                                $('.truncate-toggle', this).each(function() {
+                                    if (this.textContent === 'View more') {
+                                        this.textContent = 'View less';
+                                    } else {
+                                        this.textContent = 'View more';
+                                    }
+                                });
+                            }
+                        });
+                    } else {
+                        $(".truncate-toggle", this).css({ display:"none" });
+                    }
```

Comment:
* Fair enough, but basing the truncation on element height rather than text length has the advantage of maintaining consistent behavior regardless of what the font size is or how it's defined (if that's the desired behavior). I don't think having the condition in the template is practically much simpler than evaluating it at runtime, respective of each criterion, but ultimately there should be some clarification of what the original request was
* Good point, I'll move the block to `bindEvents`
* Good point about using jQuery functions, I'll update those
* I used the "view more" label because that seemed to be implied in #749. I would think an ellipsis and/or up/down icons would be enough to get the intent across. Whether nixing the "view less" option is a good idea rather depends on how long the text blocks are expected to be.